style.css

```diff
@@ -0,0 +1,320 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+}
+
+.Top {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 25px;
+}
+
+.Top a {
+  margin-left: 25px;
+  padding-left: 5px;
+  padding-top: 10px;
+}
+
+nav {
+  margin-right: 30px;
+  padding-right: 5px;
+  padding-top: 10px;
+}
+
+.griditems {
+  display: grid;
+  width: 100%;
+  align-items: center;
+  margin-right: 25px;
+  margin-left: 25px;
+  padding: 20px 30px 30px 10px;
+  grid-gap: 50px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  padding: 10px 20px 10px 20px;
+  font-family: "Poppins", sans-serif;
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-color: blue;
```

Comment:
**[REQUIRED CHANGES]**

- Please kindly remove the extra background color as this is not in the Figma design this will help you practice the act of being consistent and following standards. :pray: